language/stdlib/modules/Roles.move

```diff
@@ -259,6 +259,23 @@ module Roles {
          has_libra_root_role(account)
     }
 
+    /// Return true if `addr` is allowed to receive and send `Libra<T>` for any T
+    public fun can_hold_balance(account: &signer): bool acquires RoleId {
```

Comment:
@junkil-park: This is to address the issue with the "balances" column in the spreadsheet that you pointed out.

Reply:
LGTM!